templates/part.boardlist.php

```diff
@@ -102,12 +105,12 @@ class="ng-pristine ng-valid" ng-submit="boardCreate()">
 					<input class="edit ng-valid ng-empty"
 						   type="text" placeholder="<?php p($l->t('New board title')); ?>"
 						   autofocus-on-insert ng-model="newBoard.title" maxlength="100" ng-model-options="{ debounce: 250 }">
-					<div class="colorselect">
-						<div class="color" ng-repeat="c in ::colors"
-							 ng-style="{'background-color':'#{{ c }}'}"
-							 ng-click="selectColor(c)"
-							 ng-class="{'selected': (c == newBoard.color), 'dark': (newBoard.color | textColorFilter) === '#ffffff' }"></div>
-					</div>
+                    <div class="colorselect" ng-controller="ColorPickerController">
```

Comment:
There seems to be some issues with spacing here. Please fix

Reply:
I'm gonna look into this.